llpc/context/llpcCompiler.cpp

```diff
@@ -159,6 +160,9 @@ opt<int> ContextReuseLimit("context-reuse-limit",
 // -fatal-llvm-errors: Make all LLVM errors fatal
 opt<bool> FatalLlvmErrors("fatal-llvm-errors", cl::desc("Make all LLVM errors fatal"), init(false));
 
+// -experimental-new-pm: Use LLVM's new pass manager (experimental)
+opt<bool> LlvmNewPM("experimental-new-pm", cl::desc("Use LLVM's new pass manager (experimental)"), init(false));
```

Comment:
LlvmNewPM -> NewPassManager
"experimental-new-pm" -> -new-pass-manager
Keep the option variable identical to the option name. Try to avoid abbreviation in case it could denote something else other than the normal meaning.